docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,225 @@
+# Developing Themes for Accessibility
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Image alt text](#image-alt-text)
+- [Text accessibility](#text-accessibility)
+- [Keyboard accessibility](#keyboard-accessibility)
+- [Customization](#customization)
+- [Related resources](#related-resources)
+</div>
+
+Designing for accessibility means being inclusive to the needs of all users. This article outlines the basic accessibility requirements BigCommerce would recommend for all themes.
+
+We have included four accessibility best practices and examples that will help you develop a theme that meets the needs of the broadest range of abilities.
+
+### Prerequisites
+* [A BigCommerce store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial).
+* A store theme. (The following examples use the Cornerstone theme.)
+
+## Image alt text
+Alternative (alt) text is an image description read aloud by a screen reader. Adding alt text increases the content's accessibility to visually-impaired users. All HTML `<img>` elements should use the `alt` attribute to provide a text description of the image.
+
+**Store logo example**
+
+In Cornerstone, go to `templates/components/common/store-logo.html`. You will see the following code:
```

Comment:
This is a great example but would be better inside of a tutorial section. Just a simple example of where the alt text element is in relation to the `<img>` tag will be enough.